test/data.table.timings.jl

```diff
@@ -50,21 +50,6 @@ DMA = DataFrame(
   v3 =  Array{Union{Float64, Missing}}(rand(N))                            # numeric e.g. 23.5749
 );
 
-# DataArray version
```

Comment:
Better keep this to benchmark `Array{Union{T, Missing}}`?

Reply:
We have a benchmark for `Union{T, Missing}` in `DataFrame` named `DMA`.